src/StackExchange.Exceptional.AspNetCore/ConfigSettings.ErrorStore.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.ComponentModel;
+using System.Configuration;
+using System.Runtime.Serialization;
+
+namespace StackExchange.Exceptional
+{
+    internal partial class ConfigSettings
+    {
+        /// <summary>
+        /// The ErrorStore section of the configuration, optional and will default to a <see cref="Stores.MemoryErrorStore"/> if not specified.
+        /// </summary>
+        public ErrorStoreSettings ErrorStore { get; set; } 
+
+        /// <summary>
+        /// A settings object describing an error store.
+        /// </summary>
+        public class ErrorStoreSettings
+        {
+            /// <summary>
+            /// The type of error store to use, File, SQL, Memory, etc.
+            /// </summary>
+            [JsonProperty(Required = Required.Always)]
+            public string Type { get; set; }
+
+            /// <summary>
+            /// The path to use on file based error stores.
+            /// </summary>
+            public string Path { get; set; }
+
+            /// <summary>
+            /// The connection string to use on database based error stores.  If provided, ConnectionStringName is ignored.
+            /// </summary>
+            public string ConnectionString { get; set; }
+
+            /// <summary>
+            /// The name of the connection string to use from the application's configuration.
+            /// </summary>
+            public string ConnectionStringName { get; set; }
```

Comment:
`#if !NETSTANDARD2_0` wrap here please (I'll change all these later if we add an earlier `netstandard` build)